pyradiosky/skymodel.py

```diff
@@ -1147,6 +1184,7 @@ def __eq__(
         """Check for equality, check for future equality."""
         # Run the basic __eq__ from UVBase
         # the filters below should be removed in version 0.3.0
+        # TODO should we remove these filters or not?
```

Comment:
Looks like perhaps we should make a decision on this before merging.  Can you remind me of the arguments for/against?  Is it just that removing these warnings bumps our pyuvdata version requirement significantly?

Reply:
yes. I will change the comment to be labelled with a pyuvdata version rather than a pyradiosky version

Reply:
done!